client-tutorial/docker-compose.yml

```diff
@@ -6,7 +6,6 @@ services:
     image: mysql:8.0.15
     ports:
         - "3306:3306"
-    command: --init-file  /Users/henryhbk/gitclones/jomis_lfh/database scripts/create_demo_database.sql
     volumes:
        - ./database-scripts:/docker-entrypoint-initdb.d
```

Comment:
So mounting database-scripts as /docker-entrypoint-initdb.d causes the scripts in that directory to be run on container startup?

Reply:
Yes, that is correct! Here is a handy [link](https://github.com/docker-library/docs/blob/master/mysql/README.md#initializing-a-fresh-instance) I should have included on the PR description.

Reply:
I was kind of confused by this since it seemed that the init was name dependent which is weird for an argument you specify (like if it is hunting for a named file, then why have an argument?)